packages/contracts/contracts/arbitrum/ITokenGateway.sol

```diff
@@ -47,20 +47,20 @@ interface ITokenGateway {
     // );
 
     function outboundTransfer(
-        address _token,
-        address _to,
-        uint256 _amount,
-        uint256 _maxGas,
-        uint256 _gasPriceBid,
-        bytes calldata _data
+        address oken,
```

Comment:
nit: not sure why these got changed, but this should be `token` and the one below should be `to`